test/unit/contextRequests/blackButtonCreationRequest.json

```diff
@@ -16,13 +16,13 @@
           "type": "string"
         },
         {
-          "name": "op_status",
-          "value": "PENDING",
+          "name": "last_operation",
+          "value": "C",
           "type": "string"
         },
         {
-          "name": "last_operation",
-          "value": "C",
+          "name": "op_status",
+          "value": "PENDING",
```

Comment:
In fact, it is already documented as an issue at https://github.com/telefonicaid/iotagent-thinking-things/issues/50 :)